src/main/java/com/dnd/runus/client/vo/AppleAuthTokenRequest.java

```diff
@@ -0,0 +1,24 @@
+package com.dnd.runus.client.vo;
+
+import lombok.Builder;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+@Builder
+public record AppleAuthTokenRequest(
+    String code,
+    String client_id,
+    String client_secret,
+    String grant_type
+) {
+
+    public MultiValueMap<String, String> toMultiValueMap() {
+        MultiValueMap<String, String> params = new LinkedMultiValueMap<>();
+        params.add("code", code);
+        params.add("client_id", client_id);
+        params.add("client_secret", client_secret);
+        params.add("grant_type", grant_type);
+
+        return params;
```

Comment:
중요한 건 아닌데, 이렇게 하면 더 간략하게 적을 수 있을 것 같아요 

```suggestion
        return new LinkedMultiValueMap<>() {{
            add("code", code);
            add("client_id", client_id);
            add("client_secret", client_secret);
            add("grant_type", grant_type);
        }};
```